Sd collaboration updates (#16135)
newrelic/docs-website: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

import collaborationCommentonAnyPage from 'images/collaboration_screenshot-crop_comment-on-any-page.webp'

> You can collaborate with your teammates by integrating discussion tools into New Relic, allowing you to add comments on any page. Our collaboration tools will enable you to identify problems and find solutions in one place, moving the discussion closer to your data and streamlining your workflows, improving efficiency, and reducing MTTR. When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.

Reviewer: ```suggestion
You can collaborate with your teammates by integrating discussion tools into New Relic, allowing you to add comments on any page. Our collaboration tools enable you to identify problems and find solutions in one place, moving the discussion closer to your data and streamlining your workflows, improving efficiency, and reducing mean time to resolution (MTTR). When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
```

## Comment on any page [#comment-on-any-page]

> On the top navigation, most pages have a **Share** button and menu. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, select **Share** and then **Start a new discussion**. That comment stays with your data when you comment on a dashboard or an error in New Relic. You don't need to repeat information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.

Reviewer: ```suggestion
On the top navigation, most pages have a **Share** button and menu. If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, click **Share** and then **Start a new discussion**. That comment stays with your data when you comment on a dashboard or an error in New Relic. You don't need to repeat information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
```

Reviewer: @homelessbirds what's the difference between "Share to Slack" and "Start New Discussion"? This paragraph doesn't seem to mention the second option.

<img
  title= "Comment on any page"
  alt= "A screenshot of the UI that depicts commenting on a specific page."
  src={collaborationCommentonAnyPage}
  style={{ maxWidth: '60%'}}
/>

Pages with existing discussions will also have a new **Discussions** button in the top navigation that indicates how many different discussion threads exist on the page. If you click **Discussions**, a comment panel will open where you can view and contribute to those discussions. If any discussions are in **Open** status, the button will be green, as depicted in the screenshot above. The green highlighting calls attention to unanswered questions or unresolved issues you can contribute to. 

## At-mention your teammates and notify by email [#at-mention-your-teammates]

Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll receive an email notification. 

## Built-in screenshot sharing [#built-in-screenshots]

> Attach screenshots to your comments by clicking our integrated screenshot selection tool. Just click the "screenshot" icon, select the portion of the screen you want, and submit your comment. 

Reviewer: @homelessbirds I'm not wild about "screenshot" icon. Could you either remove the quotation marks or insert an inline image showing what that is? Will this be obvious to users?

Author: I fixed this. It is obvious to users, so I just removed the quotation marks.

## Close and reopen discussions [#close-reopen]

Once a question is answered or an issue is resolved, you can mark a discussion as "Closed" via the **More Actions** menu on the discussion thread. Your team can still add comments to the discussion, just like in a Jira ticket, and it will not automatically reopen a discussion. If you want to reopen a discussion, you can do it manually, like you closed it.

<img
  title= "Close and reopen discussions"
  alt= "A screenshot of the UI that depicts closing discussions."
  src={close_and_reopen_discussions}
  style={{ maxWidth: '60%'}}
/>

## Following discussions  [#follow-unfollow]

Whenever you comment or get mentioned, you'll automatically follow the discussion. This means you'll receive email notifications when new comments are added.

Stay in the loop with discussions:

* Look for the bell icon on the discussion to see if you're following.

> * Want to stop receiving notifications? Unfollow the discussion from the "More Actions" menu.

Reviewer: ```suggestion
* Want to stop receiving notifications? Go to the **More Actions** menu and click **Unfollow**.
```

> * Not interested in a specific discussion? Don't worry; you can easily follow it manually through the same menu.

Reviewer: @homelessbirds I'm not really sure what it means to following a discussion manually. What is this bulleted point trying to say?

Author: Good callout! I fixed this.
